buildingspy/tests/MyModelicaLibrary/Resources/Scripts/skipUnitTestList.txt

```diff
@@ -0,0 +1,7 @@
+# skip some mos tests
```

Comment:
I used it locally to test whether the new function works, will write a proper test instead and use it there.